src/app/components/lineage-graph/lineage-graph.component.html

```diff
@@ -45,18 +45,37 @@
             <svg:text
                 alignment-baseline="central"
                 [attr.x]="10"
-                [attr.y]="node.dimension.height / 2"
+                [attr.y]="node.dimension.height * 0.65"
                 [attr.font-weight]="node.data.isCurrent ? '500' : 'normal'"
                 [attr.cursor]="'pointer'"
                 (click)="!node.data.isCurrent && onClickNode(node)"
             >
                 {{ node.label }}
             </svg:text>
-            <svg:g *ngIf="!node.data.isRoot" [attr.transform]="'translate(' + node.dimension.width + ', 0)'">
+            <image
+                [attr.x]="10"
+                [attr.y]="8"
+                [attr.width]="18"
+                [attr.height]="18"
+                [attr.xlink:href]="(getAccountUrl(node.data.account) | async)?.avatarUrl || defaultAvatarUrl"
+            />
+            <svg:text alignment-baseline="central" [attr.x]="35" [attr.y]="16" [attr.font-weight]="'normal'">
+                {{ node.data.account }}
+            </svg:text>
+            <svg:g [attr.transform]="'translate(' + node.dimension.width / 2 + ', 0)'">
+                <image
+                    [attr.x]="-12"
+                    [attr.y]="-12"
+                    [attr.width]="20"
+                    [attr.height]="20"
+                    [attr.href]="node.data.isRoot ? '../assets/images/public.png' : '../assets/images/private.png'"
```

Comment:
make a comment here that this is temporar - in future icon will be choosen on explicit public/private attribute in the dataset

Reply:
Added comment